### PR TITLE
Additions about target deadband

### PR DESCRIPTION
**Pull request by MioRtia**
Signed-off-by: RALAMBOTIANA MIORA <miora.ralambotiana@rte-france.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**
Additions and corrections to documentation about target deadband.

### CODE REVIEW

File: docs/iidm/model/phaseTapChanger.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ Each step of a phase tap changer has the following attributes:
 # Model
 A phase tap changer is regulating if **Regulating** is set to ```true```.
 
+The **Target Deadband** defines the margin withing which the target voltage is considered as respected for discrete regulation i.e. the target voltage is considered as respected
+if the read voltage equals `targetV±(targetDeadband/2)`.
```
Comment (annetill) [inline docs/iidm/model/phaseTapChanger.md:37]: Are you also waiting for a positive value of the target deadband?

File: docs/iidm/model/phaseTapChanger.md [hunk 2]
```diff
@@ -51,6 +54,7 @@ twoWindingsTransformer.newPhaseTapChanger()
     .setRegulating(true)
```
Comment (annetill) [inline docs/iidm/model/phaseTapChanger.md:54]: It seems that in my tests, I have to set the target deadband, the regulation mode, the regulation value before the ````.setRegulating(true)````. I have not tested with steps.

Comment (MioRtia) [reply docs/iidm/model/phaseTapChanger.md:54]: In adders, the order does not matter.

File: docs/iidm/model/phaseTapChanger.md (hunk 1 above)
Comment (MioRtia) [reply docs/iidm/model/phaseTapChanger.md:37]: Yes, I will specify it.

File: docs/iidm/model/phaseTapChanger.md (hunk 2 above)
Comment (geofjamg) [reply docs/iidm/model/phaseTapChanger.md:54]: @annetill is right there is a bug in the adder implementation we need to define the deadband before the regulating to true which is not conform to "builder pattern"

**Comment by annetill** [comment]
This PR is okay for me, I have removed the WIP flag. Do you need time for other imrpovements ?

**Pull request merged**